pygmt/src/contour.py

```diff
@@ -31,11 +31,17 @@
     W="pen",
     X="xshift",
     Y="yshift",
+    b="binary",
     c="panel",
+    d="nodata",
+    e="find",
+    f="coltypes",
+    h="header",
     i="incols",
     l="label",
     p="perspective",
     t="transparency",
+    U="timestamp",
```

Comment:
Can you please put the U="timestamp" alphabetical with the capital letters? This is just for code readability, since use_alias sorts the items.

Reply:
Done, thanks @meghanrjones !